### PR TITLE
add kas-network-proxy to prow.k8s.io

### PR DESCRIPTION
**Pull request by Sh4d1**
Signed-off-by: Patrik Cyvoct <patrik@ptrk.io>

### CODE REVIEW

**Comment by cblecker** [comment]
/unassign @krzyzacy 
/uncc @chases2 
/assign @andrewsykim 
/approve
/hold

Holding for review from @andrewsykim.

**Comment by Sh4d1** [comment]
@cblecker is it required to have the test in test grid? If so, where should this one be?

**Comment by cblecker** [comment]
dashboard definition can be created here: https://github.com/kubernetes/test-infra/tree/master/config/testgrids/kubernetes/sig-cloud-provider

then add an annotation similar to this one: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/cloud-provider-vsphere/cloud-provider-vsphere-config.yaml#L43

**Comment by Sh4d1** [comment]
@cblecker done!

File: config/jobs/kubernetes-sigs/apiserver-network-proxy/apiserver-network-proxy-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    decorate: true
+    spec:
+      containers:
+      - image: golang:1.12
+        command:
+        - make
+        args:
+        - test
+    annotations:
+      testgrid-dashboards: sig-cloudprovider-apiserver-network-proxy
```
Comment (justaugustus) [inline config/jobs/kubernetes-sigs/apiserver-network-proxy/apiserver-network-proxy-presubmits.yaml]:
```suggestion
      testgrid-dashboards: sig-cloud-provider-apiserver-network-proxy
```

To match https://github.com/kubernetes/test-infra/pull/15011#pullrequestreview-309663164.

**Pull request closed without merging**